scripts/generate_bibtex.py

```diff
@@ -3,17 +3,29 @@
 import pathlib
 import warnings
 import pedpy
+import time
 
 zenodo_path = pathlib.Path("docs/source/ZENODO.rst")
 
 search_query = "PedPy"
-version = f"v{pedpy.__version__}"
-
+# version = f"v{pedpy.__version__}"
+version = "v1.1.0"
```

Comment:
why is this now hardcoded?

Reply:
This is for testing and demonstrating purposes, will be changed when testing is done.
Good catch btw!